exporters/otlp/transform_spans.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
I have been grouping these transformations in an `internal/transforms` sub-package.  I liked it because the conversion calls are of the form `transform.Mertric(...)` or `transform.MetricDesecriptor` ...

What are your thoughts on this internal package kind of approach?

Reply:
opened #527  to track this.